adapters/bombcrypto/index.js

```diff
@@ -0,0 +1,129 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BSC = require("../../sdk/EVMC");
```

Comment:
Use `binance.js` SDK instead of `EVMC.js`. If you have any issues with node, please open a support ticket in Discord.

Reply:
No problem I can change this, but bazaar the other implementation int the repo uses EVMC

Reply:
Implemented now.